src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getManufacturerStatement =
+                         connection.prepareStatement(selectManufacturer)) {
+            getManufacturerStatement.setLong(1, id);
+            getManufacturerStatement.executeQuery();
+            ResultSet resultSet = getManufacturerStatement.getResultSet();
+            List<Manufacturer> manufacturerList = unpackResultSet(resultSet);
+            if (!manufacturerList.isEmpty()) {
+                return Optional.of(manufacturerList.get(0));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer by id=" + id, e);
+        }
+        return Optional.empty();
+    }
+
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturers;
+        String selectAllManufacturers =
+                "SELECT * FROM manufacturers WHERE is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getAllManufacturersStatement = connection
+                         .prepareStatement(selectAllManufacturers)) {
+            ResultSet resultSet = getAllManufacturersStatement.executeQuery();
+            allManufacturers = unpackResultSet(resultSet);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all the data from DB", e);
+        }
+        return allManufacturers;
+    }
+
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateManufacturer =
+                "UPDATE manufacturers SET name = ?, country = ? "
+                        + "WHERE id = ? and is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateManufacturerStatement = connection
+                         .prepareStatement(updateManufacturer)) {
+            updateManufacturerStatement.setString(1, manufacturer.getName());
+            updateManufacturerStatement.setString(2, manufacturer.getCountry());
+            updateManufacturerStatement.setLong(3, manufacturer.getId());
+            int i = updateManufacturerStatement.executeUpdate();
+            if (i > 0) {
+                return manufacturer;
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer " + manufacturer, e);
+        }
+        return null;
+    }
+
+    public boolean delete(Long id) {
+        String deleteManufacturer =
+                "UPDATE manufacturers SET is_deleted = TRUE WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getManufacturerStatement =
+                         connection.prepareStatement(deleteManufacturer)) {
+            getManufacturerStatement.setLong(1, id);
+            return getManufacturerStatement.executeUpdate() >= 1;
+
```

Comment:
```suggestion
            return getManufacturerStatement.executeUpdate() > 0;
```
it's better to compare with zero